readme.md

```diff
@@ -111,19 +111,19 @@ $ npm -v
 
 * **Question:** What gets logged? Why?
 
-* **Your Answer:**
+* **Your Answer:** My name.
```

Comment:
Why?

Reply:
Answered the why part of the question and updated my readme file on my fork.